R/examine-constraints.R

```diff
@@ -90,11 +90,23 @@ print.dm_examine_constraints <- function(x, ...) {
 }
 
 kind_to_long <- function(kind) {
-  if_else(kind == "PK", "primary key", "foreign key")
+  case_when(
+    kind == "PK" ~ "primary key",
+    kind == "UK" ~ "unique key",
+    .default = "foreign key"
+  )
 }
 
 check_pk_constraints <- function(dm, progress = NA, top_level_fun = NULL) {
-  pks <- dm_get_all_pks_impl(dm)
+  pks <- dm_get_all_pks_impl(dm) %>%
+    mutate(kind = "PK") %>%
+    bind_rows(
```

Comment:
`bind_rows(PK = pks, UK = uks, .id = "kind")` ?

Reply:
sadly, `.id = "kind"` doesn't take the info from the existing column "kind", but instead creates a new one with values "1" or "2". Can still make use of it though.

Reply:
Also with named arguments?

Reply:
"1" and "2" seem dangerous.

Reply:
one step ahead of me, there, nice trick!